src/test/java/seedu/duke/data/EditCommandTest.java

```diff
@@ -0,0 +1,64 @@
+package seedu.duke.data;
+
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import seedu.duke.commands.Command;
+import seedu.duke.commands.CommandResult;
+import seedu.duke.commands.EditCommand;
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+public class EditCommandTest {
+
+    ShoppingList items = new ShoppingList();
+    Command command = new Command();
+
+
+    @BeforeEach
+    void setUp() {
+        items.add(new Item("apple", 2.0));
+        items.add(new Item("donut", 3.0));
+        items.add(new Item("banana", 4.50));
+        command.setData(items, null);
+    }
+
+    @Test
+    void testEdit_ItemDescriptionOnly_Success() {
+
+        Command command = new EditCommand(1, "apples", null);
```

Comment:
I think command.setData(items,null) should be included here because now the command is referring to a different command object (EditCommand)

Reply:
Ok thanks! :) done that